sdk/messaging/azservicebus/client.go

```diff
@@ -71,6 +71,10 @@ func NewClientFromConnectionString(connectionString string, options *ClientOptio
 	}, options)
 }
 
+// Next overloads (ie, credential sticks with the client)
```

Comment:
Do you still need this?

Reply:
Yeah, those'll be coming in soon enough.